mmdet/models/detectors/kd.py

```diff
@@ -15,23 +15,25 @@ def __init__(self,
                  bbox_head,
                  teacher_config='',
                  teacher_model='',
+                 eval_teacher=True,
                  train_cfg=None,
                  test_cfg=None,
                  pretrained=None):
         super().__init__(backbone, neck, bbox_head, train_cfg, test_cfg,
                          pretrained)
 
+        self.eval_teacher = eval_teacher
+        device = next(self.parameters()).device  # model device
+
         from mmdet.apis.inference import init_detector
 
         try:
```

Comment:
I think the config for teacher model is necessary. No need to use warnings to replace Error.

Reply:
What about warnings for teacher checkpoint, should I keep it? Because sometimes people might need the teacher to train from scratch.